src/Form/FieldCheckbox.js

```diff
@@ -87,13 +87,19 @@ class FieldCheckbox extends FieldRadioButton {
       attributes.indicatorClass
     );
 
+    const htmlAttributes = Util.pick(attributes, [
```

Comment:
I'd prefer having `pick` over `exclude`. Allowing what is for sure ok to allow is more strict and less error prone.